gef.py

```diff
@@ -5485,7 +5485,7 @@ def set_fs(uc, addr):    return set_msr(uc, FSMSR, addr)
 """
 
 
-        content = """#!/usr/bin/python -i
+        content = """#!/usr/bin/bin/env -S python -i
```

Comment:
`env -S` doesn't exist on (at least) Ubuntu 18.04 LTS which is the most used distro for GEF. Please adjust.

Reply:
Looks like coreutils is too old. I could switch to using the `which()` function to check which python to put in the shebang.

Reply:
Yep, `which()` would be an acceptable solution.

Reply:
done.